packages/components/src/components/Tabs/styles/tabs.module.scss

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+@import '../../../utilities/sizeToken.module.scss';
+@import '../../../utilities/transitions.module.scss';
+@import '../../../utilities/focusStyle.module.scss';
+
+$radius-without-border: calc(var(--radius) - var(--line-width));
+
+.root {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.triggerList {
+    position: relative;
+    flex-shrink: 0;
+    display: flex;
+    gap: sizeToken(1);
+    height: sizeToken(12);
+    flex: 1;
+    min-width: 0;
+    overflow-x: hidden;
+    &[data-size='large'] {
```

Comment:
to avoid the override of height we could perhaps assign the default height also to its respective data attribute

Reply:
what would be the benefit of that?